src/main/java/core/basesyntax/service/activity/TypeActivity.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.activity;
+
+public enum TypeActivity {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private final String name;
+
+    TypeActivity(String activityTypeLetter) {
+        this.name = activityTypeLetter;
+    }
```

Comment:
According to Mentor's review of my solution, field and parameter names should be the same.